src/Altinn.Apps/AppTemplates/AspNet/App/Startup.cs

```diff
@@ -105,6 +106,10 @@ public void ConfigureServices(IServiceCollection services)
 
             services.ConfigureDataProtection();
 
+            // Configure Azure KeyVault services
+            services.AddSingleton<IKeyVault, KeyVaultAppSI>();
```

Comment:
This line together with the new line below can be copied into an extension method.